platform/mellanox/mlnx-platform-api/sonic_platform/device_data.py

```diff
@@ -0,0 +1,134 @@
+        }
+    },
+    'ACS-MSN2100': {
+        'thermal': {
+            'minimum_table': {
+                "p2c_trust":  {"-127:120":12},
+                "p2c_untrust": {"-127:15":12, "16:25":13, "26:30":14, "31:35":15, "36:120":16},
+                "c2p_trust":   {"-127:40":12, "41:120":13},
+                "c2p_untrust": {"-127:40":12, "41:120":13},
+                "unk_trust":   {"-127:40":12, "41:120":13},
+                "unk_untrust": {"-127:15":12, "16:25":13, "26:30":14, "31:35":15, "36:120":16}
+            }
+        }
+    },
+    'ACS-MSN2010': {
+        'thermal': {
+            'minimum_table': {
+                "p2c_trust":   {"-127:120":12},
+                "p2c_untrust": {"-127:15":12, "16:20":13, "21:30":14, "31:35":15, "36:120":16},
+                "c2p_trust":   {"-127:120":12},
+                "c2p_untrust": {"-127:20":12, "21:25":13 , "26:30":14, "31:35":15, "36:120":16},
+                "unk_trust":   {"-127:120":12},
+                "unk_untrust": {"-127:15":12, "16:20":13 , "21:30":14, "31:35":15, "36:120":16}
+            }
+        }
+    },
+    'ACS-MSN3700': {
+        'thermal': {
+            'minimum_table': {
+                "p2c_trust":   {"-127:25":12, "26:40":13 , "41:120":14},
+                "p2c_untrust": {"-127:15":12, "16:30":13 , "31:35":14, "36:40":15, "41:120":16},
+                "c2p_trust":   {"-127:25":12, "26:40":13 , "41:120":14},
+                "c2p_untrust":   {"-127:25":12, "26:40":13 , "41:120":14},
+                "unk_trust":   {"-127:25":12, "26:40":13 , "41:120":14},
+                "unk_untrust": {"-127:15":12, "16:30":13 , "31:35":14, "36:40":15, "41:120":16},
+            }
+        }
+    },
+    'ACS-MSN3800': {
+        'thermal': {
+            'minimum_table': {
+                "p2c_trust":   {"-127:35":12, "36:120":13},
+                "p2c_untrust": {"-127:0":12, "1:10":13 , "11:15":14, "16:20":15, "21:35":16, "36:120":17},
+                "c2p_trust":   {"-127:30":12, "31:40":13 , "41:120":14},
+                "c2p_untrust":   {"-127:20":12, "21:30":13 , "31:35":14, "36:40":15, "41:120":16},
+                "unk_trust":   {"-127:30":12, "31:40":13 , "41:120":14},
+                "unk_untrust": {"-127:0":12, "1:10":13 , "11:15":14, "16:20":15, "21:35":16, "36:120":17},
+            }
+        }
+    },
+    'Mellanox-SN3800-D112C8': {
+        'thermal': {
+            'minimum_table': {
+                "p2c_trust":   {"-127:35":12, "36:120":13},
+                "p2c_untrust": {"-127:0":12, "1:10":13 , "11:15":14, "16:20":15, "21:35":16, "36:120":17},
+                "c2p_trust":   {"-127:30":12, "31:40":13 , "41:120":14},
+                "c2p_untrust":   {"-127:20":12, "21:30":13 , "31:35":14, "36:40":15, "41:120":16},
+                "unk_trust":   {"-127:30":12, "31:40":13 , "41:120":14},
+                "unk_untrust": {"-127:0":12, "1:10":13 , "11:15":14, "16:20":15, "21:35":16, "36:120":17},
+            }
+        }
+    },
+}
```

Comment:
You are missing SN4700 which is also supported.

Reply:
We don't have minimum table for 4700.